docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -4,50 +4,180 @@
 Risk Exposure Template
 ----------------------
 
-.. todo::
-  Add instructions, following Nathaniel's cause model template instructions. Flesh out all headings.
+.. important::
+
+   To begin documenting a risk exposure in GBD 2017 for a Vivarium simulation, start by
+   following these steps (after you have :ref:`created a new git branch
+   <contributing>` to work in):
+
+   #. Make a subdirectory :file:`{risk_exposure_name}/` in the folder
+      :file:`gbd2017/risks/` , where :file:`{risk_exposure_name}` is replaced 
+      with the (potentially shortened) name of the risk exposure you are 
+      modeling, such as :file:`unsafe_water_source/` or 
+      :file:`low_birthweight_short_gestation/`.  This subdirectory is where you 
+      will put all the files for your exposure model documentation, including 
+      this document, image files, .csv files, etc.
+
+
+   #. Copy this template into your subdirectory and rename
+      it :code:`index.rst`.
+
+   #. Replace the `internal hyperlink target
+      <https://docutils.sourceforge.io/docs/user/rst/quickref.html#internal-hyperlink-targets>`_
+      :code:`.. _2017_risk_exposure_template:` at the top of this file with a
+      unique reference label for your cause. The reference label should have the
+      form :samp:`.. _2017_exposure_{\{exposure_name\}}:`, where
+      :samp:`{\{exposure_name\}}` is replaced with a unique descriptive name or
+      abbreviation for your risk exposure, e.g.
+      :code:`.. _2017_risk_exposure_unsafe_water:`.

```

Comment:
Looks like some of these got changed to "risk_exposure" and some are still just "exposure."